deploy/kiali/kiali_cr.yaml

```diff
@@ -505,6 +505,24 @@ spec:
 #      in_cluster_url: "http://grafana.istio-system:3000"
 #      url: ""
 #
+# **Health Configuration**
+#  ---
+#  health_config:
+#
+#
+# rate: A list of configurations health that Kiali apply. Each item contains:
+#   namespace: The name of the namespace to apply. Kiali match the value like a regular expression
+#   kind: The type of the resource to apply. Kiali match the value like a regular expression
+#   name: Tha name of the resource to apply. Kiali match the value like a regular expression
+#   tolerance: A list of tolerances for this configuration. Each item contains:
+#     protocol: The protocol to apply the tolerance (grpc | http | ...) Kiali match the value like a regular expression
+#     direction: The direction to apply the tolerance (inbound | outbound | ...).Kiali match the value like a regular expression
+#     code: The status code to apply the tolerance. Kiali match the value like a regular expression
+#     degraded: The value in % to show a degraded status
+#     failure: The value in % to show a failure status,
+#    ---
+#    rate: []
+#
```

Comment:
This looks to be in the wrong spot. Is this "health_config" part (a child of) `external_services`? Because that's where this is showing it to be.

This was a brand new top-level group. It belongs here, right before the top-level `identity` section: https://github.com/kiali/kiali-operator/blob/v1.23.0/deploy/kiali/kiali_cr.yaml#L612